snaps/reference/resources.md

```diff
@@ -0,0 +1,73 @@
+---
+description: See more Snaps resources.
```

Comment:
There's nothing wrong with this, I think I am just nit-picking

```suggestion
description:Explore additional Snaps resources.
```
or 

```suggestion
description: Access more Snaps-related materials.
```

Reply:
I think I'll leave the original wording for now, because (1) the descriptions for all the reference pages start with "See" and (2) I think it might be the simplest way to word the description, and we aim for simplicity / minimal wordiness in all of our docs as part of our writing style.